packages/e2e-tests/kit-node/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "dev": "vite dev",
-    "build": "vite build",
+    "dev": "svelte-kit sync && vite dev",
```

Comment:
without this the server log could contain a warning "no .svelte-kit/tsconfig.json" found. Not needed in users projects but this simple call avoids it